### DISCUSSION TITLE
helm uninstall and install persistance

### DISCUSSION THREAD

**Discussion by vincentDcmps** [requarks/wiki#5028] [Help / Questions]
Hello,

I begin in kubernetes and helm,
so I have install wiki.js with a helm
to test database persistance I have uninstall my helm release to try to install again and see if pod work from scratch
but wiki.js pod can't connect to database anymore.
 I supose that kubectl secret for postgress password have been delete and create with a diferent password. 
what I need to do to do a reinstall with an persistant database?

**Comment by Robnety** [comment]
I workaround this by after I installed release again - helm install my-release -n wiki -f values.yaml requarks/wiki
Note - I inserted all necessary DB, etc. values into the values.yaml.

After that I did create secret where I hold only DB password:
kubectl create secret generic wiki-postgresql -n wiki
Key: postgresql-password
Value: [DB PASS]

And then I set it in helm char template:
helm upgrade my-release requarks/wiki -n wiki --reuse-values --set postgresql.existingSecret="wiki-postgresql"

This will update chart template

Hope it helped.
